centersliderview/src/main/res/values/colors.xml

```diff
@@ -1,4 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="super_dark">@android:color/black</color>
```

Comment:
can remove this empty resource file.

Reply:
will do; I guess we can re-add if there's ever a reason to have a private `colors.xml` file